activestorage/lib/active_storage/attached/many.rb

```diff
@@ -44,20 +44,25 @@ def detach
       attachments.destroy_all if attached?
     end
 
+    ##
+    # :method purge
+    #
+    # :call-seq: User.highlights.purge
+    #
     # Directly purges each associated attachment (i.e. destroys the blobs and
     # attachments and deletes the files on the service).
-    def purge
-      if attached?
-        attachments.each(&:purge)
-        attachments.reload
-      end
-    end
+    #
+    # Attachments can be selectively purged using queries
+    # e.g. User.highlights.where(id: []).purge
```

Comment:
if we want to show an example we don't need the `e.g.` and only to ident the code

Reply:
Would it be better to remove the example from the code and add it to the guide instead?